boltz/ws.go

```diff
@@ -162,6 +164,13 @@ func (boltz *Websocket) Connect() error {
 		}
 	}()
 
+	if len(boltz.swapIds) > 0 {
+		// Clear swapIds to avoid duplicates but still resubscribe to the same swaps.
```

Comment:
Wouldn't using a `map[string]bool` instead of a slice solve that problem in a nicer way?

Reply:
or just a private version of Subscribe which doesnt append to the swapIds